dev-tools/compose/nginx_ingress.conf

```diff
@@ -37,6 +37,14 @@ server {
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
 
+    location ^~ /protocol-dashboard {
+        resolver 127.0.0.11 valid=30s;
+        rewrite ^/protocol-dashboard(?!/)(.*)$ /protocol-dashboard/$1 permanent;
```

Comment:
so `/protocol-dashboard...` is rewritten to `/protocol-dashboard/...` (the actual route)